app/dashboard/views/domain_detail.py

```diff
@@ -141,6 +141,14 @@ def domain_detail(custom_domain_id):
             return redirect(
                 url_for("dashboard.domain_detail", custom_domain_id=custom_domain.id)
             )
+        elif request.form.get("form-name") == "set-name":
+            custom_domain.name = request.form.get("alias-name")
+            db.session.commit()
+            flash(f"Default alias name for Domain {custom_domain.domain} has been set", "success")
```

Comment:
this line is too long and will fail `black` check. You can run `black .` to format all files